src/test/resources/Homework18.XML

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="homework18">
+
+    <test name="play a song test" preserve-order="false">
+        <classes>
+            <class name="homework18">
+                <methods>
+                    <include name="Test"/>
```

Comment:
By the way, In homework18 test script not include method name Test -> you can remove it